README.md

```diff
@@ -53,8 +52,47 @@ Tern gives you a deeper understanding of your container's bill of materials so y
 
 # Getting Started<a name="getting-started"/>
 
+## Getting Started on Linux<a name="getting-started-on-linux">
+If you have a Linux OS you will need a distro with a kernel version >= 4.0 (Ubuntu 16.04 or newer or Fedora 25 or newer are good selections) and will need to install the following requirements:
+
+- Git (Installation instructions can be found here: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- attr (sudo apt-get install attr or sudo dnf install attr)
+- Python 3.6 or newer (sudo apt-get install python3.6(3.7) or sudo dnf install python36(37))
+- Pip (sudo apt-get install python3-pip).
+
+Some distro versions have all of these except `attr` preinstalled but `attr` is a common utility and is available via the package manager.
+
+For Docker containers
+- Docker CE (Installation instructions can be found here: https://docs.docker.com/engine/installation/#server)
+
+Make sure the docker daemon is running.
+
+Create a python3 virtual environment:
+```
+$ python3 -m venv ternenv
+$ cd ternenv
+```
+
+*NOTE:* Your OS might distribute each Python version separately. For example, on Ubuntu LTS, Python 2.7 is linked to `python2` and Python 3.6 is linked to `python3`. I develop with Python 3.7 which is installed separately with no symlinks. In this case, I use the binary. The binaries are usually installed in `/usr/bin/python`.
+
+Activate the virtual environment:
+```
+$ source bin/activate
+```
+NOTE: This specific activate script only works for Bash shells. If you need to activate a Fish Shell or C Shell you should use `source bin/activate.fish` or `source bin/activate.csh`, respectively.
+
+Install tern:
+```
+$ pip install tern
+```
+
+Run Tern:
+```
+$ tern -l report -o output.txt -i debian:buster
+```
+
 ## Getting Started with Docker<a name="getting-started-with-docker">
-Note MAC users: currently running Tern natively as a docker container is unsupported. An option is to launch a linux virtual machine (VM) on your MAC and from within follow the steps below.
+Note Mac users: currently running Tern natively as a docker container is unsupported. An option is to launch a linux virtual machine (VM) on your Mac and from within follow the steps below.
```

Comment:
Might be a good idea to redirect Mac users to the Vagrant option rather than "launch a linux virtual machine".